docs/project/linux-performance-tracing.md

```diff
@@ -48,6 +48,14 @@ Follow these steps to prepare your machine to collect a performance trace.
 1. Have two shell windows available - one for controlling tracing, referred to as **[Trace]**, and one for running the application, referred to as **[App]**.
 2. **[App]** Setup the application shell - this enables tracing configuration inside of CoreCLR.
 
+    > ```bash
+	> export COMPlus_PerfMapEnabled=1
+	> export COMPlus_EnableEventLog=1
+	> ```
```

Comment:
```suggestion
    > ```bash
	> export COMPlus_PerfMapEnabled=1
	> export COMPlus_EnableEventLog=1
	> ```
```
This is duplicated after the Note. One of the copies should be deleted.